sos/report/plugins/networking.py

```diff
@@ -82,7 +82,8 @@ def setup(self):
 
         self.add_cmd_output("ip -o addr", root_symlink="ip_addr",
                             tags='ip_addr')
-        self.add_cmd_output("route -n", root_symlink="route", tags='route')
+        self.add_cmd_output("ip route show table all", root_symlink="ip_route",
+                            tags='ip_route')
```

Comment:
We add one additional tag to this command output (https://github.com/sosreport/sos/blob/9206c4614a5844340b96b2416b19a5f7d2f91402/sos/report/plugins/networking.py#L256); both tags are properly added in a generated `manifest.json`. We might merge the tags association but I think it is redundant.

Reply:
Just changed so that we define all tags in a single place